src/libraries/externals.csproj

```
@@ -9,7 +9,7 @@
     <BinPlaceNative>true</BinPlaceNative>
     <BinPlaceRuntime>false</BinPlaceRuntime>
     <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
-    <UseLiveBuiltDotNetHost Condition="'$(TargetArchitecture)' == 's390x' or '$(TargetArchitecture)' == 'armv6'">true</UseLiveBuiltDotNetHost>
+    <UseLiveBuiltDotNetHost Condition="'$(TargetArchitecture)' == 's390x' or '$(TargetArchitecture)' == 'armv6' or '$(RuntimeOS)' == 'linux-bionic'">true</UseLiveBuiltDotNetHost>
```

We are trying to (get away from legacy `RuntimeOS` and `__DistroRid` and) converge things on two things: either it is "identifier to restore (most of the) packages - `PackageRID`" or "identifier to create shipping packages - `OutputRid`" (latter seldom differs from former in cases like non-portable builds). In this case, this condition maybe better:
```suggestion
    <UseLiveBuiltDotNetHost Condition="'$(TargetArchitecture)' == 's390x' or '$(TargetArchitecture)' == 'armv6' or $(OutputRid.StartsWith('linux-bionic'))">true</UseLiveBuiltDotNetHost>
```

---

Where is that model used within the repo? I was following the existing use of `'$(RuntimeOS)' == 'linux-musl'`

---

> Where is that model used within the repo?

See line 44 below as an example.

In the effort to simplify number of ways to alter host/target info from command-line, we consolidated and cleaned up these properties. RuntimeOS == X conditions used to be all over the place in .NET Core 3.1 and before, we have combined all that and cleaned it up from various places. We use PackageRID or OutputRid as much as possible. The reason is we want to compute the host/target info once, in one place, for the entire build. Also, RuntimeOS comes from "outside" which is not the first preference; we want to determine the host and target by ourselves and reduce the support of number of overlapping command-line options which change the build.